docs/docs/reference/rill-iso-extensions.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Rill extensions to ISO 8601
+description: All the extensions options to the standard ISO 8601 duration standard
+sidebar_label: Rill extensions to ISO 8601
+sidebar_position: 13
+---
+
+We have extended the ISO 8601 standard to specify ranges like `Week-to-Date`.
+
+### Extensions
+
+| Rill ISO extension | Description      | As Time Range | As Time Comparison |
+|--------------------|------------------|---------------|--------------------|
+| inf                | All time         | Yes           | No                 |
+| rill-TD            | Today            | Yes           | No                 |
+| rill-WTD           | Week to Date     | Yes           | No                 |
+| rill-MTD           | Month to Date    | Yes           | No                 |
+| rill-QTD           | Quarter to Date  | Yes           | No                 |
+| rill-YTD           | Year to Date     | Yes           | No                 |
+| rill-YTD           | Year to Date     | Yes           | No                 |
+| rill-PP            | Last Period      | No            | Yes                |
```

Comment:
should we call this Previous Period instead of Last Period?

Reply:
Hmmm the label in UI says `Last Period` So we can either switch to `rill-LP` and `Last Period` for label. Or `rill-PP` and `Previous Period`

Reply:
ok, then lets keep the current one.